inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,79 +1,399 @@
+** *Selecting a user from the dropdown* in the project settings section
+*Users*, you can check/unchek the *user‘s rights *(they are different
+from roles and count for this project only) on the right side. Any
+combination is possible and the user will always have _all_ rights
+combined.
+
+[cols=",,",]
+|===
+|User right |Description |*Access to Dashborad Sections *
+
+|Annotator |- annotate only a|
+- Annotation
+
+- Knowledge Base
+
+|Courator |- curate only a|
+- Couration
+
+- Monitoring
+
+- Agreement
+
+- Evaluation
+
+|Project Manager |- annotate +
+- courate +
+- create projects +
+- add new documents +
+- add guidelines +
+- manage users +
+- open documents like an other user sees it (read only) |All pages
+available
+|===
+
+* In the section **Layers, **you may modify existing layers to use for
+your annotation and create own layers. In case you do not want to work
+on default layers only, we highly recommend reading the main
+documentation on layers. LINK EINFÜGEN
+* In the section **Knowledge Base, **you can change the settings for the
+knowledge bases used in your project and also import local and remote
+knowledgebases or create a new one. Herefore, just click the
+„create“-button and follow the wizzard.
+** Note that you can have several knowledge bases in your INCEpTION
+instance but you can choose for every project which one(s) to use. Using
+many knowledge bases in one project will slow down the performance.
+** Via the *Dashboard* (click the Dashboard-button at the top centre),
+you can get to the knowledge base page. This is a page different from
+the one in the project settings where you can modify and work on your
+knowledge bases.
+** For details on knowledge bases, see our main documentation LINK
+EINFÜGEN or our tutorial video „Overview“ which also talks about
+knowledge bases quickly:
+https://www.youtube.com/watch?v=wp4AN3p23mQ&list=PL5Hz5pttaj96SlXHGRZf8KzlYvpVHIoL-&index=3&t=0s..
+If you like to explore a knowledge base learning by doing, you may
+download and import the example project _Interactive Concept Linking_:
+https://inception-project.github.io/example-projects/concept-linking/
+which contains a small knowledge base.
+* In the section **Recommenders, **you can create and modify your
+recommenders. They learn from your behaviour and give automatic
+suggestions for creating annotations. For details on this topic to broad
+for a getting-started-guide, see our main documentation LINK EINFÜGEN or
+our tutorial video „Recommender Basics“:
+https://www.youtube.com/watch?v=Xz3Hs8Lyoeg&list=PL5Hz5pttaj96SlXHGRZf8KzlYvpVHIoL-&index=3
+.
+
+* In the *Guidelines* section, you may import guideline files for your
+team to read and to check quickly while working. On the annotation page
+(→ _dashboard_ → _annotation_ → open any document), everyone can quickly
+look them up by clicking on the guidelines button on the top (it looks
+like a book).
+
+* We have imported a whole project already (the example project) as well
```

Comment:
I would remove this sentence or say
INCEpTION can not only import projects, but also export them...